### DISCUSSION TITLE
Vite dev server and .vue files

### DISCUSSION THREAD

**Discussion by kreha1** [cypress-io/cypress#15950] [Component Testing]
Hi, I've managed to get vite dev server running, but I have no idea how to actually make it work with vue files (and also typescript).
```
> src/main.ts:3:16: error: No loader is configured for ".vue" files: src/App.vue
    3 │ import App from '@/App.vue';
      ╵                 ~~~~~~~~~~~

Error: Build failed with 1 error:
src/main.ts:3:16: error: No loader is configured for ".vue" files: src/App.vue
```
I'm a bit lost to be honest - I thought that if vite manages all this (with plugins etc), and this dev server is using that very same config, shouldn't it work out of the box? What am I missing or misunderstanding?

**Comment by lmiller1990** [comment]
Hey! Sorry for the slow reply. Vite *does* all the heavy lifting. This should work. Can you share your repo? Where did you configure this alias? I'll give this a try in the meantime.

Also, have you got `vite.config.ts` with the vue plugin (assuming so)?
